src/wp-includes/global-styles-and-settings.php

```diff
@@ -317,5 +312,6 @@ function wp_theme_has_theme_json() {
  */
 function wp_clean_theme_json_cache() {
 	wp_cache_delete( 'wp_get_global_stylesheet', 'theme_json' );
+	wp_cache_delete( 'wp_get_global_styles_svg_filters', 'theme_json' );
```

Comment:
This needs to be refreshed against `trunk` to resolve a new merge conflict :)

Reply:
FIxed in https://github.com/WordPress/wordpress-develop/pull/3926/commits/5448f9cbd2d07620829437346aae973de112dc33